src/lib.rs

```diff
@@ -53,14 +53,14 @@ impl vm::Ext for EwasmExt {
 
     /// Returns a value for given key.
     fn storage_at(&self, key: &H256) -> Result<H256> {
-        let ret = ewasm_api::storage_load(&Bytes32 { bytes: key.0 });
+        let ret = ewasm_api::storage_load(&Bytes32::from(key.0));
         // FIXME: why isn't there a From trait for converting between [u8;32] and H256?
-        Ok(H256::from(ret.bytes))
+        Ok(H256::from(Into::<[u8; 32]>::into(ret)))
```

Comment:
You could also have a variable:
```
let ret = easm_api::storage_load(&Bytes32::from(key.0));
let ret:[u8;32] = ret.into();
Ok(H256::from(ret))
```

Reply:
> Weird it cannot find the common denominator in cases like this.

It could be that I've missed something.

> You could also have a variable:

That's cleaner, will update